### ISSUE TITLE
Hyperparameters training

### ISSUE THREAD

**Issue by rifaadev** [jsyoon0823/GAIN#31]
Hello, How I can use GAIN with tensorboard hparms to get the best hyperparameters for a specifies dataset ?

**Comment by jsyoon0823** [comment]
Sorry. I have not used the hyper-parameter tuning with Tensorboard.

**Comment by rifaadev** [comment]
Thank you for your answer.  How you choose the Hyperparameters ?

**Comment by jsyoon0823** [comment]
You can do some random search or greedy search to find the best hyper-parameters in terms of accuracy.
